tests/unit/amo/components/TestReportAbuseButton.js

```diff
@@ -25,13 +27,15 @@ describe(__filename, () => {
     ...props
   } = {}) {
     return mount(
-      <ReportAbuseButton
-        addon={addon}
-        errorHandler={errorHandler}
-        i18n={getFakeI18nInst()}
-        store={store}
-        {...props}
-      />
+      <I18nProvider i18n={getFakeI18nInst()}>
+        <ReportAbuseButton
+          addon={addon}
+          errorHandler={errorHandler}
+          i18n={getFakeI18nInst()}
```

Comment:
We could maybe extract a variable here since `getFakeI18nInst()` is called twice.